core/trino-main/src/main/java/io/trino/sql/planner/ExpressionInterpreter.java

```diff
@@ -439,6 +439,15 @@ protected Object visitIsNullPredicate(IsNullPredicate node, Object context)
                 return new IsNullPredicate(toExpression(value, type(node.getValue())));
             }
 
+            if (value instanceof SqlRow sqlRow) {
```

Comment:
This also needs to be fixed in the code generator (io.trino.sql.gen.IsNullCodeGenerator) and IrExpressionInterpreter, and for IsNotNullPredicate in all places.